monte_carlo.ipynb

```diff
@@ -39,7 +39,12 @@
     "numberOfCustomerEvents = round(1.1*timeHorizon/meanTimeBetweenCustomerArrival)\n",
     "prices = np.arange(0,15.1,0.1)\n",
     "watchedCompany = 0\n",
-    "numberSimulations = 1000"
+    "numberSimulations = 1000\n",
+    "numberConsideredMarketSituations = 10\n",
+    "m = 2\n",
+    "numberCustomerEvents = 4\n",
+    "simulatedHorizon = 1\n",
+    "reactionTimeCompany = timeHorizon"
```

Comment:
Before writing in a language, always consult its styling guide (reactionTimeCompany > reaction_time_company). ;)
Or at least Google's. Next time ...

Reply:
Will do that. The variable names are subject to change anyway.